static/app/components/feedback/feedbackSearch.tsx

```diff
@@ -37,6 +37,10 @@ const EXCLUDED_TAGS: string[] = [
   FieldKey.PLATFORM,
 ];
 
+const isTagExcluded = (tagKey: string): boolean => {
+  return EXCLUDED_TAGS.includes(tagKey) || tagKey.startsWith('ai_categorization.label.');
```

Comment:
why not add to `EXCLUDED_TAGS` list?

Reply:
I think it's easier to read, understand, and grep when the things are just laid out explicitly like this. 

The issue lists only had one item in them to start, so clearly someone intended for the list to grow.